### PR TITLE
API:Blocks & API:SetPageLanguage

### PR DESCRIPTION
**Pull request by NikitaRana07**
Modified README.md to include information and links for API: Blocks & API: SetPageLanguage.
Modified modules.json 
Added code sample files for  
1. API: Blocks 
2. API:SetPageLanguage

### CODE REVIEW

File: modules.json [hunk 1]
```diff
@@ -366,5 +366,30 @@
...
+			"action": "query",
+			"format": "json",
+			"list": "blocks",
+			"bklimit": "3",
+			"bkprop": "id|user|by|timestamp|expiry|reason|range|flags"
+		}	
+	
+	},
+	{
+		"filename": "setlanguage.py",
```
Comment (srish) [inline modules.json]: Parameters related to `setpagelanguage` module will not go in the `modules.json` file. Auto-generating code samples process currently support only those modules that are `GET` only.

File: modules.json [hunk 2]
```diff
@@ -366,5 +366,30 @@
...
+		"docstring": "Demo of `SetPageLanguage` module: POST request to change the language of a page",
+		"endpoint": "https://en.wikipedia.org/w/api.php",
+		"params": {
+			"action": "setpagelanguage",
+			"format": "json",
+			"pageid": "123",
+			"lang": "eu",
+			"token": "e17ac3bb8d7dd0c4f5e4fd1f84cd21095c76c2cc+\\"
```
Comment (srish) [inline modules.json]: Information like tokens is sensitive information and one should never hard-code them in the code like you've added it here :) 

You will be programmatically obtaining a token and writing somewhat similar to `edit` module code for `setPageLanguage` module: https://github.com/wikimedia/MediaWiki-Action-API-Code-Samples/blob/master/python/edit.py

File: python/blocks.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/python3
+
+"""
+    blocks.py
```
Comment (srish) [inline python/blocks.py]: It does not look like you've auto-generated this file. Please see: https://github.com/wikimedia/MediaWiki-Action-API-Code-Samples#contributing-code-samples.

Comment (NikitaRana07) [reply python/blocks.py]: I did autogenerated this file. When I saw the other pages you improved none of the codes there had that autogenerated comment, so I deleted it, if you wish I can edit it to include the comment.

**Comment by NikitaRana07** [comment]
Thanks a lot for your valuable feedback! I will make the changes and get them back to you tomorrow. I have already made the documentation pages, please take a look. I appreciate your help!

File: modules.json (hunk 1 above)
Comment (NikitaRana07) [reply modules.json]: Sorry my bad!

File: modules.json (hunk 2 above)
Comment (NikitaRana07) [reply modules.json]: okay, yeah I thought so! thanks a lot for the feedback! :)

**Comment by NikitaRana07** [comment]
I made all the changes you asked for.
(i) modified modules.json.
(ii) Modified setlanguage.py 
(iii)Modified blocks.py
The API documentation is also ready  
API:Blocks
https://www.mediawiki.org/wiki/User:Nikitrain/Sandbox/API:Blocks
API:SetPageLanguage
https://www.mediawiki.org/wiki/User:Nikitrain/Sandbox/API:SetPageLanguage
I have also checked the codes for any indentation error.

**Comment by srish** [comment]
@NikitaRana07 Thanks a lot for your work on this! I've made minor fixes to documentation here:
https://www.mediawiki.org/wiki/User:Nikitrain/Sandbox/API:SetPageLanguage
https://www.mediawiki.org/wiki/User:Nikitrain/Sandbox/API:Blocks

You can copy paste the changes to MediaWiki.org. When you publish your changes make sure to say in the comment: "updated as per the improving the action API documentation project". For reference check following page's history as it recently got updated: https://www.mediawiki.org/w/index.php?title=API:Move&action=history. Let me know if you've any trouble while doing so :)

**Comment by NikitaRana07** [comment]
Thanks for the review and those changes! you are the best! I have :)
I copied the changes, take a look!
https://www.mediawiki.org/wiki/API:SetPageLanguage
https://www.mediawiki.org/wiki/API:Blocks
would love to hear some feedback!

**Pull request merged**